_translator-files/po/de/Running-a-Server.po

```diff
@@ -445,219 +483,223 @@ msgid ""
-msgstr ""
+msgstr "Hier wird der Pfad festgelegt, in dem die Aufzeichnungen des Servers gespeichert werden sollen. Wenn dieser Pfad festgelegt ist, startet die Funktion \"Jam-Recorder aktivieren\" auf der Registerkarte \"Server-Setup\" die Aufzeichnung, sobald sich die erste Person mit dem Server verbindet, und stoppt, wenn die letzte Person den Server verlässt. Verwende die Schaltfläche \"Neue Aufnahme\", um ein neues Unterverzeichnis zu erstellen, in dem die Aufnahmen von nun an gespeichert werden sollen. Beachten, dass die Aufnahmen pro Spur im Format [Audacity](https://www.audacityteam.org/) `.lof` und [REAPER](https://en.wikipedia.org/wiki/REAPER) `.rpp` vorliegen. Öffne die entsprechenden Dateien, um die Aufnahmen in diesen Anwendungen anzuhören."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:227
 #, no-wrap
 msgid "**Note**: When your Server is recording, Clients will display a message that recording is on.\n"
-msgstr ""
+msgstr "**Hinweis**: Wenn dein Server aufzeichnet, zeigen die Clients eine Meldung an, dass die Aufzeichnung läuft.\n"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:229
 msgid "### Custom Directory Address"
-msgstr ""
+msgstr "### Benutzerdefinierte Verzeichnisadresse"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:231
 msgid "Leave this field empty unless you need to list your Server on a [Custom Directory](#3-directory) or run a Directory."
-msgstr ""
+msgstr "Lasse dieses Feld leer, es sei denn, du möchtest deinen Server in einem [benutzerdefinierten Verzeichnis](#3-verzeichnis-server) auflisten oder ein Verzeichnis betreiben."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:233
 msgid "### Server List Filename"
-msgstr ""
+msgstr "### Verzeichnis Server - Server Liste"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:235
 msgid "Leave this field empty unless you need to run your Server as a [Directory](#3-directory). When in use, this holds the list of registered Servers whilst restarting the Directory. This prevents the server list appearing \"empty\" until the Servers re-register."
-msgstr ""
+msgstr "Lass dieses Feld leer, es sei denn, du möchtest deinen Server als [Verzeichnis Server](#3-verzeichnis-server) betreiben. Wenn das Feld benutzt wird, wird die Liste der registrierten Server angezeigt, während das Verzeichnis neu gestartet wird. Dies verhindert, dass die Server Liste \"leer\" erscheint, bis sich die Server neu registrieren."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:237
 msgid "### Delay panning"
-msgstr ""
+msgstr "### Delay Panning - Stereo Effekt"
```

Comment:
It's called verzögertes Panning in the german App translation.

Reply:
Corrected